controllers/login.js

```diff
@@ -20,11 +21,32 @@ router.post('/login', (req, res) => {
         res.status(200);
         return res.json(data[0]);
     })
-    .catch(err => { res.sendStatus(500) });
+    .catch(err => { 
+        res.sendStatus(500) 
+        logger.error(err);
+    });
 });
 
 router.post('/sign-up', (req, res) => {
-    res.send('/sign-up working');
+    const username = req.body.username;
```

Comment:
Just wanted to make you aware of this possibility: https://jsfiddle.net/gpaqrtnc/
You can leave it like you have it, it's totally fine but just giving another option.  It's called object destructuring.  Here's a link if you're interested: https://tylermcginnis.com/object-array-destructuring/